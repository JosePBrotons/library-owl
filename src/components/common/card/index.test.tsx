import React from 'react';
import renderer from 'react-test-renderer';
import Card from '.';

test('renders card component correctly', () => {
    const cardCmp = renderer.create(<Card />).toJSON();
    expect(cardCmp).toMatchSnapshot();
});
