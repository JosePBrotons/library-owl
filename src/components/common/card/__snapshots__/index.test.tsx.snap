// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders login component correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(255, 255, 255, 1)",
      "borderRadius": 5,
      "elevation": 3,
      "flex": 1,
      "marginVertical": 10,
      "padding": 15,
      "shadowColor": "rgba(69, 69, 69, 0.5)",
      "shadowOffset": Object {
        "height": 2,
        "width": 2,
      },
      "shadowOpacity": 0.5,
      "shadowRadius": 3,
      "width": "100%",
    }
  }
/>
`;
